MEME/MEME/Model/ModelSearchView/ModelSearchViewController.swift

```diff
@@ -0,0 +1,272 @@
+//
+//  ModelSearchViewController.swift
+//  MEME
+//
+//  Created by 정민지 on 1/15/24.
+//
+
+import UIKit
+import SwiftUI
+
+class ModelSearchViewController: UIViewController {
+    //MARK: -Properties
+    let backButton: UIButton = {
+        let backButton = UIButton()
+        backButton.setTitle("<", for: .normal)
+        backButton.setTitleColor(UIColor.black, for: .normal)
+        backButton.backgroundColor = UIColor.white
```

Comment:
```suggestion
        backButton.backgroundColor =.white
```
잘 아시겠지만 위와 같이 사용할 수도 있습니다 :)